docs/core/testing/unit-testing-fsharp-with-mstest.md

```diff
@@ -4,12 +4,8 @@ description: Learn unit test concepts for F# in .NET Core through an interactive
 author: billwagner
 ms.author: wiwagn
 ms.date: 08/30/2017
-ms.topic: conceptual
 dev_langs: 
   - "fsharp"
```

Comment:
Left this devlang for F#. but the next file you remove it

Reply:
I didn't touch (or not supposed to touch) any of dev_langs.